src/main/java/org/benetech/servicenet/config/FilterConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package org.benetech.servicenet.config;
+
+import org.benetech.servicenet.filters.RequestFilter;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FilterConfiguration {
+
+    @Bean
+    public RequestFilter requestFilter() {
+        return new RequestFilter();
+    }
+
+    @Bean
+    public FilterRegistrationBean <RequestFilter> filterRegistrationBean() {
+        FilterRegistrationBean <RequestFilter> registrationBean = new FilterRegistrationBean();
+
+        registrationBean.setFilter(requestFilter());
+        registrationBean.addUrlPatterns("/api/*");
```

Comment:
I wonder if it wouldn't be better to put this filter in gateway service, so that requests from all the other services would also be logged. I think there are also a couple of endpoints having different root url - i believe /management is one of them

Reply:
Done